kuwala/core/backend/app/dbt/kuwala_blocks/macros/utils/get_result_query.sql

```diff
@@ -0,0 +1,14 @@
+{% macro get_result_query(block_columns, query) %}
+    {% set block_columns_string = get_columns_string(block_columns) %}
+
+    {% set result %}
+        -- KUWALA_TRANSFORMATION_START
+        SELECT {{ block_columns_string }}
+        FROM ({{ query }}) AS result
+        -- KUWALA_TRANSFORMATION_END
```

Comment:
shouldn't this comment removed? i saw in first few sql file the previous comments are removed.